api/envoy/config/trace/v3/opencensus.proto

```diff
@@ -48,29 +48,35 @@ message OpenCensusConfig {
   reserved 7;
```

Comment:
@htuch you may know why - the one thing i didnt manage to do was deprecate the enum above (in file)

not sure why - probs i was just doing it wrong

Reply:
It probably only matters that you deprecate the fields that reference it.